src/HonzaBotner.Discord.Services/Commands/ReminderCommands.cs

```diff
@@ -35,16 +36,18 @@ public ReminderCommands(IRemindersService service, IOptions<ReminderOptions> opt
         [GroupCommand]
         public async Task Create(
             CommandContext context,
-            [Description("Date or time of the reminder")] string rawDatetime,
-            [Description("Title of the reminder.")] string title,
-            [Description("Optional additional content"), RemainingText] string? content = null
+            [Description("Date or time of the reminder")]
+            string rawDatetime,
+            [Description("Title of the reminder."), RemainingText]
+            string title
         )
         {
-            var now = DateTime.Now;
-            var datetime = ParseDateTime(rawDatetime);
+            DateTime now = DateTime.Now;
+            DateTime? datetime = ParseDateTime(rawDatetime);
```

Comment:
Why use explicit type when using `var` on lines 69, 71, ... This imo makes the codebase slightly inconsistent